src/backend/web/authentication.js

```diff
@@ -70,7 +71,7 @@ function init() {
     done(null, user);
   });
   passport.deserializeUser(function (user, done) {
-    done(null, user);
+    done(null, new User(user.name, user.email, user.id));
```

Comment:
Once `user.id` has been refactored as above (to be initialized using a hashed email address), remove `user.id` from this call to construct a new User.

Reply:
They have to keep it in, since we're pulling it off the session for use in later code.